packages/synapse-react-client/src/utils/hooks/useDetectSSOCode.ts

```diff
@@ -33,6 +32,8 @@ export type UseDetectSSOCodeOptions = {
     resp: TwoFactorAuthErrorResponse,
     encodedTwoFaResetToken: string,
   ) => void
+  hasInitializedSession?: boolean
+  token?: string
```

Comment:
I am not convinced we need this over context, but if we do, add a comment to clarify that it's not an OAuth provider-related token
```suggestion
  /* Synapse access token, if the user is currently logged in */
  token?: string
```